applications/Unity.GrantManager/src/Unity.GrantManager.Domain/Settings/GrantManagerSettingDefinitionProvider.cs

```diff
@@ -6,7 +6,8 @@ public class GrantManagerSettingDefinitionProvider : SettingDefinitionProvider
 {
     public override void Define(ISettingDefinitionContext context)
     {
-        //Define your own settings here. Example:
-        //context.Add(new SettingDefinition(GrantManagerSettings.MySetting1));
+        context.Add(
+            new SettingDefinition("SectorFilter","",null,null,isVisibleToClients:true,isInherited:false,isEncrypted:false).WithProviders(TenantSettingValueProvider.ProviderName)
```

Comment:
can we please complete the nulls here please, think these are L resources if i look at the default setting definitions

Reply:
pushed now a new code